src/main/java/edu/harvard/hms/dbmi/bd2k/irct/IRCTApplication.java

```diff
@@ -288,24 +308,21 @@ public void addJoin(String name, IRCTJoin join) {
 	 * @param resultDataConverters
 	 *            the resultDataConverters to set
 	 */
-	public void setResultDataConverters(
-			Map<ResultDataType, List<DataConverterImplementation>> resultDataConverters) {
+	public void setResultDataConverters(Map<ResultDataType, List<DataConverterImplementation>> resultDataConverters) {
 		this.resultDataConverters = resultDataConverters;
 	}
 
 	/**
 	 * Returns a dataconveter for a given datatype and format
-	 * 
+	 *
 	 * @param dataType
 	 *            DataType
 	 * @param format
 	 *            Format
 	 * @return DataConverter
 	 */
-	public ResultDataConverter getResultDataConverter(ResultDataType dataType,
-			String format) {
-		List<DataConverterImplementation> dciList = this.resultDataConverters
-				.get(dataType);
+	public ResultDataConverter getResultDataConverter(ResultDataType dataType, String format) {
```

Comment:
I am just using Eclipse settings. Good luck getting everybody set their Eclipse(like) IDE to match your asthetics!